jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -762,7 +762,7 @@ void setPresenceExtensions(Collection<ExtensionElement> extensions)
                 return;
             }
 
-            lastPresenceSent = lastPresenceSent.cloneWithNewId();
+            lastPresenceSent = lastPresenceSent.asBuilder((String) null).build();
```

Comment:
Since Smack is trying to move to immutable Stanzas, my theory was to have MucClient store a PresenceBuilder rather than a Presence for lastPresenceSent.  This is why I contributed the PR to add StanzaBuilder.removeExtension in Smack 4.4.4.

Reply:
Yes, that's really the better idea. Thanks!